sdks/python/apache_beam/transforms/window_test.py

```diff
@@ -149,7 +146,7 @@ def test_sessions(self):
     result = (pcoll
               | 'w' >> WindowInto(Sessions(10))
               | GroupByKey()
-              | sort_values
+              | Map(lambda (k, vs): (k, sorted(vs)))
```

Comment:
Could you explain this a little bit more? What is causing the recursion? Does it affect how future tests should be written?